### ISSUE TITLE
Alternative text missing from images on the Guide to Topics page

### ISSUE THREAD

**Issue by axamei** [pulibrary/special_collections#525]
https://rbsc.princeton.edu/topics

**Comment by sandbergja** [comment]
These images now have an alt text if it is supplied, and an empty alt attribute if not.  Closing this ticket as completed.
